samples/nrf9160/aws_fota/README.rst

```diff
@@ -0,0 +1,67 @@
+.. _mqtt_simple_sample:
```

Comment:
copy paste? (whole file)

Reply:
yes

Reply:
This is mostly a place holder file for documentation.